tests/accounts/test_accounts_profile.py

```diff
@@ -103,10 +103,27 @@ def test_profile_name_field(accounts_base_url, selenium):
 
 
 @test_case('C195551')
-@expected_failure
 @accounts
 def test_profile_username_field(accounts_base_url, selenium):
     """Test the user's username field."""
+    # setup
+    page = Profile(selenium, accounts_base_url).open()
+    username = os.getenv('STUDENT_USER')
+    password = os.getenv('STUDENT_PASSWORD')
+    page.log_in(username, password)
+    assert (page.logged_in), 'User is not logged in'
+    # at profile, store original values
+    old_username = page.username.username
+    new_username = Utility.random_hex()
+    # set new values
+    page.username.username = new_username
+    assert (page.username.username != old_username), 'Username change failed'
+    # reset the fields to the original values
+    from time import sleep
+    sleep(0.25)
+    page.username.username = old_username
+    sleep(0.25)
+    assert (page.username.username == old_username), 'Username reset failed'
```

Comment:
If snooze time is needed, this should happen in the model (either a `wait.until` for a field to show up or an explicit `sleep` while waiting for an animation/transition to complete).